utilities/works-pdf/src/utils/auditLogs.js

```diff
@@ -0,0 +1,39 @@
+const get = require("lodash.get");
+const { create_audit_details } = require("../api");
+
+async function createEgPaymentsExcelLog(requestData, values) {
+    await createAuditLogs(requestData, "CREATE", {
+        paymentId, 
+        paymentNumber, 
+        "tenantId": requestData.payment.tenantId, 
+        "status": "INPROGRESS",
+        "lastmodifiedby": userid,
+        "lastmodifiedtime": new Date().getTime()
+    })
+}
+
+async function createAuditLogs(requestData, operation, values){
+    let RequestInfo = requestData.RequestInfo;
```

Comment:
@SumelRattan-eGov add logs while creating the audit logs

Reply:
Added logs and try-catch